tests/pr_test.go

```diff
@@ -177,6 +177,31 @@ func TestRunStandardSolutionSchematics(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 }
 
+func TestRunStandardUpgradeSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
```

Comment:
There is no reason to lock into us-south here - we should use `BestRegionYAMLPath: regionSelectionPath` to generate the region - I think we locked all our tests into us-south and that should not be the case - we dont want to overload that region.

NOTE: the code comment that says `For FSCloud locking into us-south since that is where the HPCS permanent instance is` is NOT correct - can you remove it and update the other tests too please?